app/app/calculations/models.py

```diff
@@ -61,7 +60,7 @@ class Punching(models.Model):
 
     # geometry
     support = models.ForeignKey(Support)
-    sect = models.ForeignKey(Sect)
+    sect = models.ForeignKey(Section)
```

Comment:
Zapomniałam tu zmienić, chyba trzeba poprawić to na section Występuje też w innych miejscach.